c/zss.c

```diff
@@ -389,6 +394,25 @@ static JsonObject *readServerSettings(ShortLivedHeap *slh, const char *filename)
   return mvdSettingsJsonObject;
 }
 
+static void initZssBackgroundTasks(HttpServer *server) {
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG,"initZssBackgroundTasks\n");  
+  // register background handler
+  STCBase *base = server->base;
+  stcRegisterModule(
+    base,
+    STC_MODULE_ZSS,
+    server,
+    NULL,
+    NULL,
+    NULL,
+    processZssBackgroundHandler
+  );
+
```

Comment:
@ifakhrutdinov - here is our registration to STC Module
we call `stcRegisterModule`, with new module constant `STC_MODULE_ZSS` to process background handler `processZssBackgroundHandler`